### PR TITLE
Force PEP

### PR DESCRIPTION
**Pull request by singlerider**
@KellerFuchs 's commits will appear to go away after all is merged with `master`.

```bash
find . -name '*.py' -exec autopep8 --in-place '{}' \;
```

#codestandardsmatter

### CODE REVIEW

**Review by KellerFuchs — DISMISSED** [review]
This will conflict with #29

**Comment by singlerider** [comment]
Yeah, don't merge now. I'll handle that and drop commits once everything is in the bag (merged).

**Pull request closed without merging**
